Documentation/project-docs/standard-platform.md

```diff
@@ -23,12 +23,14 @@ The .NET Platform Standard version represents binary portability across platform
 
 
 ## Terms
+- **.NET Platform Standard** - versioned sets of the reference assemblies that all .NET Platforms must support.
```

Comment:
We need to clarify guard rails more. The design for how it will look is incomplete. We do not want any platforms to implement just a subset so we're not sure that feature should exist.